CONTRIBUTING.rst

```diff
@@ -0,0 +1,75 @@
+Contributing to this repository
+===============================
+
+|:smile:| |:heart_eyes:| First off, thanks for taking the time to contribute! With your help, we’re making developers' lives easier. |:heart_eyes:| |:smile:|
+
+The following is a set of tips and guidelines that will help you improve the Aiven developer portal.
+
+What should I know before I contribute?
+---------------------------------------
+
+**What is devportal?**
```

Comment:
I've been writing this as "DevPortal" but I'm not sure we have an agreed standard anywhere.

Reply:
One for the automatic check?